cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -683,6 +693,7 @@ static DataprocConf create(Map<String, String> properties) {
     final String customImageUri = getString(properties, CUSTOM_IMAGE_URI);
     final String gcpCmekKeyName = getString(properties, ENCRYPTION_KEY_NAME);
     final String gcpCmekBucket = getString(properties, "gcsBucket");
```

Comment:
nit: would you mind moving this to a constant for consistency?

Reply:
Done.